Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Use Ubuntu 16.04 LTS as the basis for the Docker image.
+
+FROM ubuntu:20.04
+
+# Set timezone:
+RUN ln -snf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && echo $CONTAINER_TIMEZONE > /etc/timezone
+
+# Install all the Linux packages required for Yocto / Toradex BSP builds. Note that the packages python3,
+# tar, locales and cpio are not listed in the official Yocto / Toradex BSP documentation. The build, however, fails
+# without them. curl is used for brining in the repo tool. repo tool uses git, so thats being instaled here aswell.
+
+RUN apt-get update && apt-get -y install gawk wget git-core diffstat unzip texinfo gcc-multilib \
+     build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
+     xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev \
+     pylint3 xterm tar locales curl git
+RUN apt-get -y install software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get -y update && \
+    apt-get -y install python3.6
+
+# By default, Ubuntu uses dash as an alias for sh. Dash does not support the source command
+# needed for setting up the build environment in CMD. Use bash as an alias for sh.
+RUN rm /bin/sh && ln -s bash /bin/sh
+
+# Set the locale to en_US.UTF-8, because the Yocto build fails without any locale set.
+RUN locale-gen en_US.UTF-8 && update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+ENV USER_NAME ot3
```

Comment:
let's go ahead and change this to `opentrons` or something. It probably doesn't matter, but still.